node_modules/oae-jitsi/lib/api.meetings.js

```diff
@@ -0,0 +1,690 @@
+var _ = require('underscore');
+
+var AuthzAPI = require('oae-authz');
+var AuthzConstants = require('oae-authz/lib/constants').AuthzConstants;
+var AuthzInvitations = require('oae-authz/lib/invitations');
+var AuthzPermissions = require('oae-authz/lib/permissions');
+var LibraryAPI = require('oae-library');
+var log = require('oae-logger').logger('meetings-jitsi-api');
+var MessageBoxAPI = require('oae-messagebox');
+var MessageBoxConstants = require('oae-messagebox/lib/constants').MessageBoxConstants;
+var OaeUtil = require('oae-util/lib/util');
+var PrincipalsDAO = require('oae-principals/lib/internal/dao');
+var PrincipalsUtil = require('oae-principals/lib/util');
+var ResourceActions = require('oae-resource/lib/actions');
+var Signature = require('oae-util/lib/signature');
+var Validator = require('oae-authz/lib/validator').Validator;
+
+var MeetingsAPI = require('oae-jitsi');
+var MeetingsConstants = require('./constants').MeetingsConstants;
+var MeetingsDAO = require('./internal/dao');
+
+/**
+ * PUBLIC FUNCTIONS
+ */
+
+/**
+ * Create a new meeting.
+ * 
+ * @param {any} ctx
+ * @param {any} displayName
+ * @param {any} description
+ * @param {any} chat
+ * @param {any} contactList
+ * @param {any} visibility
+ * @param {any} additionalMembers
+ * @param {any} callback
+ * @returns
+ */
+var createMeeting = module.exports.createMeeting = function (ctx, displayName, description, chat, contactList, visibility, additionalMembers, callback) {
+
+    callback = callback || function() {};
+
+    // Setting content to default if no visibility setting is provided
+    visibility = visibility || Config.getValue(ctx.tenant().alias, 'visibility', 'meeting');
+
+    var allVisibilities = _.values(AuthzConstants.visibility);
+
+    // Convert chat and contactList value to boolean for validation (if there are present)
+    if (chat) {
+        if (chat === 'true') chat = true;
```

Comment:
I think this whole check should be moved down to the validator block (since we are re-doing there) and could be written more succinctly as `chat = chat === 'false' ? false : chat === 'true';`